app/src/main/java/org/p2p/wallet/rpc/repository/amount/RpcAmountInteractor.kt

```diff
@@ -0,0 +1,26 @@
+package org.p2p.wallet.rpc.repository.amount
+
+import org.koin.ext.getFullName
+import timber.log.Timber
+import java.math.BigInteger
+
+class RpcAmountInteractor(
+    private val localRepository: RpcAmountLocalRepository,
+    private val remoteRepository: RpcAmountRemoteRepository
+) {
+
+    suspend fun getFees(commitment: String?): BigInteger {
+        var cachedLamports = localRepository.getLamportsPerSignature()
+        return if (cachedLamports != null) {
+            Timber
+                .tag(RpcAmountRemoteRepository::class.getFullName())
+                .d("Getting from cache, lamportsPerSignature: $cachedLamports")
+
+            cachedLamports
+        } else {
+            val lamports = remoteRepository.getFees(commitment)
```

Comment:
how getFees returns me lamports? i thought i would get fees when calling this method :)

Reply:
It's actually fee but in lamports format

Reply:
it would be much easier to read if we had simple class Lamport / Lamports that contained primitive, but for now i suggest renaming it to getFeesInLamports?